### PR TITLE
Side

### PR DESCRIPTION
**Pull request by accel33**
Git logs.txt added

### CODE REVIEW

File: git_names.txt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+js file new
```
Comment (accel33) [inline git_names.txt]: Leaving some comment

**Comment by boatnoodles** [comment]
what are the logs file for?

**Comment by boatnoodles** [comment]
hi

**Comment by accel33** [comment]
The logs files are information send from "git logs" and "git show" ### qwerqwer **asdfasdf** _curvi_ 

> La`Ka` @boatnoodles #3

**Pull request merged**